quarkchain/cluster/master.py

```diff
@@ -1286,6 +1293,11 @@ def handle_new_root_block_header(self, header, peer):
         future_list = self.broadcast_rpc(
             op=ClusterOp.ADD_ROOT_BLOCK_REQUEST, req=AddRootBlockRequest(r_block, False)
         )
+
+        for grpc_client in self.grpc_slave_pool:
+            result = grpc_client.set_rootchain_confirmed_block()
+            self.grpc_result.append(result)
```

Comment:
I don't think we need persist the results to `self`

Reply:
it's just used for testing since we don't have return in this function

Reply:
you should find another way to test the results of the call, instead of storing them into `Master`